app/models/host/base.rb

```diff
@@ -196,6 +196,13 @@ def set_interfaces(parser)
       if !self.managed? && self.primary_interface.mac.blank? && self.primary_interface.identifier.blank?
         identifier, values = parser.suggested_primary_interface(self)
         self.primary_interface.mac = Net::Validations.normalize_mac(values[:macaddress]) if values.present?
+        if values.present?
+          # bridge interfaces are not attached to parent interface so save would not be possible
+          if interface_class(identifier) != Nic::Bridge && values[:virtual].present? && values[:attached_to].present?
+            self.primary_interface.virtual = values[:virtual]
+            self.primary_interface.attached_to = values[:attached_to]
```

Comment:
It might be also good to set the tag, `values[:tag]` is set to `1` if interface identifier is `eth0.1`. Once we provision the host, we'd correctly deploy the vlan configuration. Otherwise user need to configure the vlan id on subnet level in advance.